app/api/business/brief/brief_business.py

```diff
@@ -13,6 +16,25 @@ def is_open_to_all(brief):
     return False
 
 
+def can_submit_response_to_brief(brief, user, check_response_limit=True):
+    user_status = BriefUserStatus(brief, user)
+
+    if user_status.has_supplier_errors() or not user_status.is_approved_seller():
+        return False, 'Supplier is invalid'
```

Comment:
If I'm a seller trying to respond to an opportunity but my profile is invalid, how will I know from this message?  Should we make it clearer to the user that there's something they need to fix?

Reply:
I believe the frontend will show the fact they are invalid in their dashboard. The ability to respond using the UI should also be removed (no button to respond) and even if they navigate to the responding URL, I believe they are shown an error about the fact their profile has errors. I don't believe this error will reach a UI action but it will be worth testing that to confirm.